pkg/auth/csp/selfmanaged.go

```diff
@@ -85,7 +85,7 @@ func interruptHandler(d context.CancelFunc) {
 	signal.Notify(c, os.Interrupt)
 	for range c {
 		d()
-		log.Fatal("Login flow interrupted\n")
+		log.Fatal(errors.New("login flow interrupted"), "")
```

Comment:
Should we use the message parameter instead?
```suggestion
		log.Fatal(nil, "login flow interrupted")
```